todo_app/trello.py

```diff
@@ -0,0 +1,88 @@
+import requests
```

Comment:
Sorry I believe the pdf you were given is out of date. You should be using Poetry (as described in the README.md file, not pip.

Delete requirements.txt
And you should have added this dependency with `poetry add requests` (which will also automatically update the pyproject.toml file for you)